sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/DeploymentController.java

```diff
@@ -10,38 +15,43 @@
 import com.sap.ai.sdk.core.model.AiDeploymentDeletionResponse;
 import com.sap.ai.sdk.core.model.AiDeploymentList;
 import com.sap.ai.sdk.core.model.AiDeploymentModificationRequest;
-import com.sap.ai.sdk.core.model.AiDeploymentModificationResponse;
 import com.sap.ai.sdk.core.model.AiDeploymentTargetStatus;
 import com.sap.ai.sdk.core.model.AiParameterArgumentBinding;
 import com.sap.ai.sdk.foundationmodels.openai.OpenAiModel;
 import java.util.List;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 /** Endpoints for AI Core AiDeployment operations */
 @Slf4j
 @RestController
+@SuppressWarnings("unused")
 @RequestMapping("/deployments")
 class DeploymentController {
```

Comment:
(Comment)

Too bad we have to add this annotation.
Normally IntelliJ default settings are clever enough to identify program / service entry points.
I don't know any other solution.